Document/0x06f-Testing-Local-Authentication.md

```diff
@@ -197,8 +197,8 @@ It is important to remember that the LocalAuthentication framework is an event-b
 
 - Verify that sensitive processes, such as re-authenticating a user performing a payment transaction, are protected using the keychain services method.
 - Verify that access control flags are set for the keychain item which ensure that the keychain item its data can only be unlocked by means of authenticating the user. This can be with one of the following flags:
-  - `kSecAccessControlBiometryCurrentSet` (before iOS 11.3 `kSecAccessControlTouchIDCurrentSet`). This will make sure that a user needs to authenticate with his biometrics (e.g. Face ID or Touch ID) before he can access the data in the keychain item. Whenever the user adds a fingerprint or appearance to the device, it will automatically invalidate the entry in the Keychain. This makes sure that the keychain item can only ever be unlocked by the users that were enrolled when the item was added to the keychain.
-  - `kSecAccessControlBiometryAny` (before iOS 11.3 `kSecAccessControlTouchIDAny`). This will make sure that a user needs to authenticate with his biometrics (e.g. Face ID or Touch ID) before he can access the data in the Keychain entry. The Keychain entry will survive any (re-)enroling of new fingerprints or facial representation. This can be very practical if the user has a changing fingerprint. However, it also means that attackers, who are somehow able to enrole their fingerprints or facial representations to the device, can now access those entries as well.
+  - `kSecAccessControlBiometryCurrentSet` (before iOS 11.3 `kSecAccessControlTouchIDCurrentSet`). This will make sure that a user needs to authenticate with biometrics (e.g. Face ID or Touch ID) before accessing the data in the keychain item. Whenever the user adds a fingerprint or appearance to the device, it will automatically invalidate the entry in the Keychain. This makes sure that the keychain item can only ever be unlocked by the users that were enrolled when the item was added to the keychain.
```

Comment:
```suggestion
  - `kSecAccessControlBiometryCurrentSet` (before iOS 11.3 `kSecAccessControlTouchIDCurrentSet`). This will make sure that a user needs to authenticate with biometrics (e.g. Face ID or Touch ID) before accessing the data in the keychain item. Whenever the user adds a fingerprint or facial representation to the device, it will automatically invalidate the entry in the Keychain. This makes sure that the keychain item can only ever be unlocked by users that were enrolled when the item was added to the keychain.
```